### DISCUSSION TITLE
Can we adopt this viewer to Nvidia NGP instant NeRF?

### DISCUSSION THREAD

**Discussion by at82** [julienkay/MobileNeRF-Unity-Viewer#6] [Q&A] [answered]
Hey, very nice work! I was wondering if there was an easy way to adopt your code for importing a NeRF trained using Nvidia NGP instant NeRF implementation? Thanks!

**Comment by julienkay** [comment, marked as answer]
Hi, unfortunately there's no easy way to do this. The learned rasterizer-friendly representation is specific to MobileNeRF. Nvidia has made their own advances in that direction with [ndiffrec](https://github.com/NVlabs/nvdiffrec) and [GET3D](https://nv-tlabs.github.io/GET3D/), but those outputs will most likely be traditional meshes and not benefit from anything in this repo. Sorry.

**Comment by at82** [comment]
Thanks a lot for the quick answer!
